config/services.yaml

```diff
@@ -1,3 +1,8 @@
+parameters:
+    env(DEFAULT_LOCALE): 'en'
+    env(SERVICE_NAME): 'articles'
```

Comment:
I take it that this is a default value only

Reply:
Yep.